process/transaction/shardProcess.go

```diff
@@ -598,13 +604,49 @@ func makeUserTxFromRelayedTxV2Args(args [][]byte) *transaction.Transaction {
 	return userTx
 }
 
+func (txProc *txProcessor) finishExecutionOfRelayedTxV3(
+	relayerAcnt, acntDst state.UserAccountHandler,
+	tx *transaction.Transaction,
+	userTx *transaction.Transaction,
+) (vmcommon.ReturnCode, error) {
+	computedFees := txProc.computeRelayedTxV3Fees(tx, userTx)
+	txHash, err := txProc.processTxV3AtRelayer(
```

Comment:
this is not necessary processing, but taking out fees from relayer. maybe rename the function.

Reply:
renamed to consumeFeeFromRelayer